### PR TITLE
fix(module:date-picker): fix datePicker show multi panel

### PR DESCRIPTION
**Pull request by luckyship**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x ] The commit message follows our guidelines: https://github.com/NG-ZORRO/ng-zorro-antd/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ x] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Application (the showcase website) / infrastructure changes
[ ] Other... Please describe:
```

## What is the current behavior?

Issue Number: https://github.com/NG-ZORRO/ng-zorro-antd/issues/7450

## What is the new behavior?
it wiill show normally

## Does this PR introduce a breaking change?
```
[ ] Yes
[ x] No
```

### CODE REVIEW

**Comment by wenqi73** [comment]
This change causes another problem: the cursor in the input element cannot be moved with `mousedown`

**Comment by luckyship** [comment]
> This change causes another problem: the cursor in the input element cannot be moved with `mousedown`

@wenqi73 thanks for pointer my fault, i have filter input element when prevent default event, pls review again

File: components/date-picker/date-picker.component.ts [hunk 1]
```diff
@@ -369,6 +369,8 @@ export class NzDatePickerComponent implements OnInit, OnChanges, OnDestroy, Afte
       this.focus();
       this.updateInputWidthAndArrowLeft();
     });
+
+    this.elementRef.nativeElement.addEventListener('mousedown', this.onMouseDown);
```
Comment (wenqi73) [inline components/date-picker/date-picker.component.ts:373]: Could you add some comments about the issue to this?

Comment (luckyship) [reply components/date-picker/date-picker.component.ts:375]: ok, i have added comments for this

**Pull request merged**